### PR TITLE
chore(docs): updated docs with scss issue workaround

### PR DESCRIPTION
**Pull request by RaphaelAmadeu**
### Related Ticket(s)

#1741 

### Description

I've updated the `README` files, so they include the fix for the scss import issue

### CODE REVIEW

File: packages/styles/README.md [hunk 1]
```diff
@@ -33,6 +33,13 @@ application:
 import '@carbon/ibmdotcom-styles';
 ```
 
+For Carbon component styles to properly import, the following should be added to
+your environment variables:
+
+```
+SASS_PATH=node_modules:src
+```
```
Comment (jeffchew) [inline packages/styles/README.md]: @kennylam wondering if this is enough context for adopters or if we should provide a more detailed explanation around this, and if this is an issue on the Carbon side or not that we're trying to do a workaround for?

cc: @RaphaelAmadeu

Comment (kennylam) [reply packages/styles/README.md]: @jeffchew @RaphaelAmadeu Yes, agree a little more detail could be helpful.
 maybe something like 
```suggestion
> 💡 There may be times pathing errors are encountered when importing certain stylesheets. We're working hard to fix
> these, but in the meantime you can add the following to your `.env` file to resolve.

```
SASS_PATH=node_modules:src
```
```

Comment (RaphaelAmadeu) [reply packages/styles/README.md]: @kennylam Thanks for the explanation text, I've added it to my latest PR

File: packages/styles/README.md.rej [hunk 2]
```diff
@@ -0,0 +1,6 @@
+diff a/packages/styles/README.md b/packages/styles/README.md	(rejected hunks)
+@@ -36 +36,3 @@ import '@carbon/ibmdotcom-styles';
+-💡 There may be times pathing errors are encountered when importing certain stylesheets. We're working hard to fix these, but in the meantime you can add the following to your .env file to resolve:
++💡 There may be times pathing errors are encountered when importing certain
++stylesheets. We're working hard to fix these, but in the meantime you can add
++the following to your .env file to resolve:
```
Comment (jeffchew) [inline packages/styles/README.md.rej]: Hey @RaphaelAmadeu , what is this file?

Comment (RaphaelAmadeu) [reply packages/styles/README.md.rej]: @jeffchew , I've removed it

**Pull request merged**